.github/PULL_REQUEST_TEMPLATE/pull_request_template_berk.md

```diff
@@ -0,0 +1,86 @@
+# Guidelines
+
+## _Soft_ Schema Freeze
+
+The `nmdc-schema` and `berkeley-schema-fy24` schemas are under a soft freeze, which means changes **should not** be made that have any downstream implications. To ensure this, all PRs created creating during the freeze will be closely reviewed with **every** component of the NMDC system in mind.
+
+## Reviewers
+
+To ensure no changes are made unexpectedly, PR creators will request reviews from _all_ [Berkeley Schema Roll Out task coordinators](https://docs.google.com/document/d/1XXN1YuaBuSkxPXeiLKm5YxYzXTamBPQrzzeLhlh7PWs/edit#heading=h.u52g8v319adh).
+
+We expect task coordinators to review PRs and provide feedback/approval within 1 week of when they are identified as reviewers. 
+
+PRs will **NOT** be merged until all task coordinators (or one of their delegates) have approved. 
+
+Expedition, questions, and discussion can happen at any meeting.
+
+Delays in review & mergeing should be address in meetings or with NMDC Leadership.
+
+# PR Information
+
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactor
+- [ ] Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation
+- [ ] Schema change: Content
+  - slot or class name
+  - slot multiplicity changes (from a single value to a list or vice versa)
+  - any changes to the ways slots are assigned to any classes
+  - creating a new slot/Class
+  - Enum value changes
+- [ ] Schema change: Cleanup
+  - descriptions of slots, classes, or enums
+  - removal of unused, commented, or invalid code
+  - updated mappings of terms, classes, or slots to ontologies
+  - added an additional Enum to accommodate new or future metadata
+- [ ] Changes to any files or directories in `src/schema`
+     
+## Description
+
+> _PRs should be [small and concise](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/best-practices-for-pull-requests#write-small-prs)._
+>
+> Aim to create small, focused pull requests that fulfill a single purpose. Smaller pull requests are easier and faster to review and merge, leave less room to introduce bugs, and provide a clearer history of changes.
+
+- Replace this text with a description of what this PR branch contains. Please keep in mind that all reviewers will be reading this description. Example: "In this branch, I..."
+
+## Related Issues
+
+> All PRs should relate to or fix an issue(s). Please identify the issue(s) below.
```

Comment:
Ideally, the issue will appear in the right hand side of the PR screen. I think the section is called "development"

That link is created automatically when issues are linked to branches